dispatch/bin/dispatch-admin

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
This is a comment is it not? does this actually cause any sort of change to the function of the code?

Reply:
It is not a comment, even though it starts with a #. It is called a [shebang line](https://stackoverflow.com/questions/2429511/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-pyt). When you execute dispatch-admin, that line determines what interpreter will run this code.